packages/react-linked-input/LinkedInput.js

```diff
@@ -10,7 +10,7 @@
 'use strict';
 
 var React = require('react');
-var LinkedValueUtils = require('react/lib/LinkedValueUtils');
+var LinkedValueUtils = require('react-dom/lib/LinkedValueUtils');
```

Comment:
clone-with-props isn't a thing anymore, don't worry about it.

It's definitely a breaking change but not to a part of an API contract we've published. There will almost certainly be some packages that stop working.

I guess we could update this package in good faith, so we can leave this.